navig/MsgOdometry odometry

navig/MsgPoint2d center
int32 pixel_size
float64 size_ratio

navig/MsgObjectPosition pos

float64 depth

int32 frame_number